docs/operator-guide/troubleshooting/troubleshoot-container-registries.md

```diff
@@ -0,0 +1,25 @@
+# Container Registry Troubleshooting
+
+## Problem
+
+Reset container registry is not feasible due to the `Change registry` button not working.
+
+## Cause
+
+The External Secrets Operator (ESO) doesn't allow you to reconfigure registry.
```

Comment:
The KubeRocketCI Portal does not allow to reconfigure the registry because the registry secrets have external owners.